src/main/java/AndreeaG/utils/fileUtils.java

```diff
@@ -0,0 +1,26 @@
+package AndreeaG.utils;
+
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import java.io.File;
+import java.io.IOException;
+
+public class fileUtils {
```

Comment:
Name of the class is with upperCase letter. I was referring to the name of the package AndreeaG -> andreeag

Reply:
aha, okay, I think I got it